GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -48,6 +48,7 @@ public interface IGitTfsRemote
         string Prefix { get; }
         bool ExportMetadatas { get; set; }
         Dictionary<string, string> ExportWorkitemsMapping { get; set; }
+        int? InitialChangeset { get; set; }
```

Comment:
I agree - I think I'd gone with the property out of sheer frustration and desire to just expose the needed information. It should have been a method pair.